terraform/deployments/govuk-publishing-infrastructure/app_assets_s3.tf

```diff
@@ -2,8 +2,12 @@ resource "aws_s3_bucket" "app_assets" {
   bucket        = "govuk-app-assets-${var.govuk_environment}"
   force_destroy = var.force_destroy
   tags = {
+    Product     = "GOV.UK"
+    System      = "App assets"
+    Service     = "App assets S3"
```

Comment:
I think we should keep the Service description a bit more general and not duplicate the name of the system. The guidance says:
> Service: used to describe the function of a particular resource (for example: account management, session storage, front end)

What do you think of describing it as `S3` or `Object storage`

Reply:
Object storage probably best